include/swift/AST/Module.h

```diff
@@ -337,6 +337,18 @@ class ModuleDecl : public DeclContext, public TypeDecl {
     Bits.ModuleDecl.IsSystemModule = flag;
   }
 
+  /// Returns true if this module is a non-Swift module that was imported into
+  /// Swift.
+  ///
+  /// Right now that's just Clang modules.
+  bool isNonSwiftModule() const {
```

Comment:
I'm planning follow-up work where I check on all uses of `findUnderlyingClangModule()` that don't actually need to check more than what this predicate is doing.

Reply:
Oof, `ModuleDecl::isClangModule` already exists and I just didn't spot it. This is still a better implementation, but maybe I'll give up on my too-forward-looking name.

Reply:
…and the one use of `isClangModule` is incorrect, because it looks through overlays.